back/config/locales/en.yml

```diff
@@ -285,6 +285,7 @@ en:
       likes_count: 'Likes'
       location: 'Location'
       longitude: 'Longitude'
+      neutral_reactions_count: 'Unsure'
```

Comment:
I think this meaning is not good. I would stick with 'Neutral' as it is clearer, more flexible and also matches with the naming in the code (a bonus)

'Unsure' is the opposite of 'sure', but I think what we want here is actually 'neutral' == no strong opinion (not a 'like' or 'dislike', not an 'upvote' or 'downvote', etc.

In other words, I might be sure (certain) that my evaluation is 'neutral'. I do not think an idea or position is good or bad.